C/sort/shaker_sort.c

```diff
@@ -0,0 +1,61 @@
+#include<stdio.h>
+void shakersort(int [],int);
+void swap(int *,int *);
+int main()
+{
+    int num,i,arr[10000];
+    printf("Enter the number of elements:");
+    scanf("%d",&num);
+    for(i=0;i<num;i++)
+               {       scanf("%d",&arr[i]);
+                      }
+    printf("Enter the elements to be sorted:");
+    for(i=0;i<num;i++)
+              {        printf("%d ",arr[i]);
+                      }
+    shakersort(&arr[0],num);
+    printf("\n");
+    printf("Sorted Array:");
+    for(i=0;i<num;i++)
+             {         printf("%d ",arr[i]);
+                      }
+    getch();
+    return 0;
+}
+void shakersort(int arr[],int num)
+{
+     int num1,num2,i;
+     num1=num-1;
+     num2=1;
+     // Bidirectional comparison
+     while(num2<=num/2)
+      {  for(i=num2-1;i<num1;i++)
+                {           if(arr[i]>arr[i+1])
+                                              swap(&arr[i],&arr[i+1]);
+                                              }
+         num1=num1-1;
+         for(i=num1;i>=num2;i--)
+                 {          if(arr[i]<arr[i-1])
+                                              swap(&arr[i],&arr[i-1]);
+                                              }
+         num2=num2+1;
+      }
```

Comment:
Why is the indentation so mis-matched? Please use a proper indentation pattern across the repo